docs/README.md

```diff
@@ -4,3 +4,15 @@ This directory contains documentation on:
 
 - Using the SDK as a consumer
 - Implementing new event formats and protocol bindings
+
+## Implementation utility classes
+
+The `CloudNative.CloudEvents.Impl` namespace contains utility
```

Comment:
What about Core instead of Impl?

Reply:
`Core` sounds like something that *would* be worth importing for a normal user. `Impl` isn't right either though... let's both keep thinking about it, and see if we can come up with something better. Glad you're okay with the general approach though :)

Reply:
Moved into Core now as discussed in meeting. Will merge on green.